src/ints/int10_memory.cpp

```diff
@@ -22,112 +22,179 @@
+	                                      (uint16_t)(offset * 32));
+
+	uint16_t base = real_readw(BIOSMEM_SEG, BIOSMEM_CRTC_ADDRESS);
+
+	bool mono = (base == VGAREG_MDA_CRTC_ADDRESS);
+
+	// Put video adapter in planar mode
+	IO_Write(0x3c4, 0x02);
+
+	// select plane 2 for writing
+	IO_Write(0x3c5, 0x04);
+	IO_Write(0x3c4, 0x04);
+
+	// odd/even off in SEQ
+	IO_Write(0x3c5, 0x07);
+	IO_Write(0x3ce, 0x04);
 
-void INT10_LoadFont(PhysPt font,bool reload,Bitu count,Bitu offset,Bitu map,Bitu height) {
-	PhysPt ftwhere=PhysicalMake(0xa000,map_offset[map & 0x7]+(uint16_t)(offset*32));
-	uint16_t base=real_readw(BIOSMEM_SEG,BIOSMEM_CRTC_ADDRESS);
-	bool mono=(base==VGAREG_MDA_CRTC_ADDRESS);
+	// select plane 2 for reading
+	IO_Write(0x3cf, 0x02);
+	IO_Write(0x3ce, 0x05);
 
-	//Put video adapter in planar mode
-	IO_Write(0x3c4,0x02);IO_Write(0x3c5,0x04); // select plane 2 for writing
-	IO_Write(0x3c4,0x04);IO_Write(0x3c5,0x07); // odd/even off in SEQ
-	IO_Write(0x3ce,0x04);IO_Write(0x3cf,0x02); // select plane 2 for reading
-	IO_Write(0x3ce,0x05);IO_Write(0x3cf,0x00); // write mode 0, odd/even off in GFX
-	IO_Write(0x3ce,0x06);IO_Write(0x3cf,0x04); // CPU memory window A0000-AFFFF
+	// write mode 0, odd/even off in GFX
+	IO_Write(0x3cf, 0x00);
+	IO_Write(0x3ce, 0x06);
 
-	//Load character patterns
-	for (Bitu i=0;i<count;i++) {
-		MEM_BlockCopy(ftwhere+i*32,font,height);
-		font+=height;
+	// CPU memory window A0000-AFFFF
+	IO_Write(0x3cf, 0x04);
+
+	// Load character patterns
+	auto font = _font;
+
+	for (auto i = 0; i < count; i++) {
+		MEM_BlockCopy(ftwhere + i * 32, font, height);
+		font += height;
 	}
-	//Load alternate character patterns
-	if (map & 0x80) {
-		while (Bitu chr=(Bitu)mem_readb(font++)) {
-			MEM_BlockCopy(ftwhere+chr*32,font,height);
-			font+=height;
+
+	// Load alternate 9x14 or 9x16 character patterns on VGA based on
+	// state of the clocking mode register
+	if (IS_VGA_ARCH && !vga.seq.clocking_mode.is_eight_dot_mode) {
+		while (auto chr = mem_readb(font++)) {
+			MEM_BlockCopy(ftwhere + chr * 32, font, height);
+			font += height;
 		}
 	}
 
-	//Return to normal text mode
-	IO_Write(0x3c4,0x02);IO_Write(0x3c5,0x03); // select planes 0&1 for writing
-	IO_Write(0x3c4,0x04);IO_Write(0x3c5,0x03); // odd/even on in SEQ
-	IO_Write(0x3ce,0x04);IO_Write(0x3cf,0x00); // select plane 0 for reading
-	IO_Write(0x3ce,0x05);IO_Write(0x3cf,0x10); // write mode 0, odd/even on in GFX
-	IO_Write(0x3ce,0x06);IO_Write(0x3cf,mono?0x0a:0x0e); // Bx000-BxFFF, odd/even on
+	// Return to normal text mode
+	IO_Write(0x3c4, 0x02);
+
+	// select planes 0&1 for writing
+	IO_Write(0x3c5, 0x03);
+	IO_Write(0x3c4, 0x04);
+
+	// odd/even on in SEQ
+	IO_Write(0x3c5, 0x03);
+	IO_Write(0x3ce, 0x04);
+
+	// select plane 0 for reading
+	IO_Write(0x3cf, 0x00);
+	IO_Write(0x3ce, 0x05);
+
+	// write mode 0, odd/even on in GFX
+	IO_Write(0x3cf, 0x10);
+	IO_Write(0x3ce, 0x06);
+
+	// Bx000-BxFFF, odd/even on
+	IO_Write(0x3cf, mono ? 0x0a : 0x0e);
```

Comment:
👨‍🍳🐔 
```suggestion
	// Return to normal text mode

	// select planes 0&1 for writing
	IO_Write(0x3c4, 0x02);
	IO_Write(0x3c5, 0x03);

	// odd/even on in SEQ
	IO_Write(0x3c4, 0x04);
	IO_Write(0x3c5, 0x03);

	// select plane 0 for reading
	IO_Write(0x3ce, 0x04);
	IO_Write(0x3cf, 0x00);

	// write mode 0, odd/even on in GFX
	IO_Write(0x3ce, 0x05);
	IO_Write(0x3cf, 0x10);

	// Bx000-BxFFF, odd/even on
	IO_Write(0x3ce, 0x06);
	IO_Write(0x3cf, mono ? 0x0a : 0x0e);
```

Reply:
Cheers, yeah I split the lines first, then moved the comments around, that was my mistake 😄